SimulationAgent/Runtime/Config.cs

```diff
@@ -46,13 +53,23 @@ public Config(IConfigData configData)
                                     "value in the 'appsettings.ini' configuration file.");
             }
 
+            var limitsConf = new RateLimitingConfiguration
+            {
+                ConnectionsPerSecond = configData.GetDouble(CONNECTIONS_FREQUENCY_LIMIT_KEY),
+                RegistryOperationsPerMinute = configData.GetDouble(REGISTRYOPS_FREQUENCY_LIMIT_KEY),
+                MessagesPerDay = configData.GetDouble(MESSAGES_FREQUENCY_LIMIT_KEY),
```

Comment:
nit: whole number/integer for daily message cap

Reply:
I'd prefer keeping all the values as real numbers for consistency and easier documentation